src/CommunityToolkit.Aspire.Hosting.Golang/PublicAPI.Shipped.txt

```diff
@@ -2,4 +2,4 @@
 Aspire.Hosting.ApplicationModel.GolangAppExecutableResource
 Aspire.Hosting.ApplicationModel.GolangAppExecutableResource.GolangAppExecutableResource(string! name, string! workingDirectory) -> void
 Aspire.Hosting.GolangAppHostingExtension
-static Aspire.Hosting.GolangAppHostingExtension.AddGolangApp(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name, string! workingDirectory, int port = 8080, string![]? args = null) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.GolangAppExecutableResource!>!
```

Comment:
This line should not be edited.
The following line should be added to `PublicAPI.Unshipped.txt` file instead :

```
*REMOVED*static Aspire.Hosting.GolangAppHostingExtension.AddGolangApp(this Aspire.Hosting.IDistributedApplicationBuilder! builder, string! name, string! workingDirectory, int port = 8080, string![]? args = null) -> Aspire.Hosting.ApplicationModel.IResourceBuilder<Aspire.Hosting.ApplicationModel.GolangAppExecutableResource!>!
```

Reply:
I always forget that